rpc/pb/rpc.proto

```diff
@@ -549,6 +559,9 @@ message Account {
 
     // frozen balance information
     repeated FrozenBalance frozen_balances = 7;
+
+    // vote information
+    repeated VoteInfo votes = 8;
```

Comment:
I think naming "vote_info" should be better, as "votes" means the number of votes above.

Reply:
vote_info or vote_infos?